website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -63,6 +63,9 @@ The table below shows this endpoint's support for
   - `OIDCClientSecret` `(string: <required>)` - The OAuth client secret
     configured with your OIDC provider.
 
+  - `OIDCDisableUserInfo` `(bool: <required>)` - Disable claims from the OIDC UserInfo
+    endpoint
```

Comment:
This flag is preventing Nomad from making a request to the external provider of user info, right?  Let's explain that in a little more detail _why_ you'd want this. Also, booleans always default to `false`.

```suggestion
  - `OIDCDisableUserInfo` `(bool: false)` - When set to `true`, Nomad will not make a request to the identity provider to get OIDC UserInfo. You may wish to set this if your identity provider...TBD
```

Reply:
Hi, @tgross!

I would suggest

> - `OIDCDisableUserInfo` `(bool: false)` - When set to `true`, Nomad will not make a request to the identity provider to get OIDC UserInfo. **You may wish to set this if your identity provider sending all necessary user claims in OIDC token**.

Should I add some links to MS FAQ?
> Microsoft OIDC provider returns only sub claim in UserInfo endpoint
And https://learn.microsoft.com/en-us/windows-server/identity/ad-fs/overview/ad-fs-faq#i-m-trying-to-get-more-claims-on-the-userinfo-endpoint--but-it-s-only-returning-subject--how-can-i-get-more-claims-

Reply:
I would not link to their FAQ, as once they re-org'ed MSDN you can't really rely on them keeping their links intact anymore.

Reply:
Ok, I've updated description comment for the history